### PR TITLE
[Test] Add test for imageseg-decoder and tensor_filter profiling

### PR DESCRIPTION
**Pull request by anyj0527**
- Add test cases for newly added option `snpe-deeplab` and `snpe-depth`
- It also check profiling option of tensor_filter

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/nnstreamer_decoder_image_segment/runTest.sh [hunk 1]
```diff
@@ -78,6 +78,26 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! videoc
 
 callCompareTest test_golden.0 test_output.0 1 "test with videotestsrc" 0
 
+# THIS WON'T FAIL, BUT NOT MUCH MEANINGFUL (TARGET MODEL IS DIFFERENT ONE)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
+videotestsrc num_buffers=1 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
+    t. ! queue ! mix. \
+    t. ! queue ! tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
+        tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} latency=1 throughput=1 ! \
```
Comment (jaeyun-jung) [inline tests/nnstreamer_decoder_image_segment/runTest.sh:86]: Q: what means 'latency=1 throughput=1'?

Comment (anyj0527) [reply tests/nnstreamer_decoder_image_segment/runTest.sh:86]: They're recently added properties for profiling tensor_filter. App may get invoke latency information from this.
Details: #2417, #2422

Comment (myungjoo) [reply tests/nnstreamer_decoder_image_segment/runTest.sh:86]: Maybe we need to rename them later: e.g., ```profile_latency=1 profile_throughput=1``` or ```profile=latency,throughput``` / ```profile=no```.

File: tests/nnstreamer_decoder_image_segment/runTest.sh [hunk 2]
```diff
@@ -78,6 +78,26 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! videoc
 
 callCompareTest test_golden.0 test_output.0 1 "test with videotestsrc" 0
 
+# THIS WON'T FAIL, BUT NOT MUCH MEANINGFUL (TARGET MODEL IS DIFFERENT ONE)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
+videotestsrc num_buffers=1 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
```
Comment (myungjoo) [inline tests/nnstreamer_decoder_image_segment/runTest.sh:83]: usually ```num_buffers```  with 3 or more is better unless you really need to do it for 1 frame only.

Comment (anyj0527) [reply tests/nnstreamer_decoder_image_segment/runTest.sh]: Right, I increase the number to 4. Thanks!

**Pull request merged**